src/components/ec-btn-dropdown/ec-btn-dropdown.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div
+    ref="popperReference"
+    class="ec-btn-dropdown"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-btn-dropdown` : 'ec-btn-dropdown'"
+  >
+    <ec-btn
+      :is-disabled="isDisabled"
+      is-reverse
+      category="primary"
+      is-rounded
+      class="ec-btn-dropdown__btn"
+      data-test="ec-btn-dropdown__btn"
+      @click="$emit('click')"
+    >
+      {{ textButton }}
+    </ec-btn>
+    <ec-dropdown-search
+      :items="itemsDropdown"
+      :is-search-enabled="false"
+      :popper-modifiers="popperModifiers"
+      :disabled="isDisabled"
+      @change="(value) => $emit('change', value)"
+      @open="isActive = true"
+      @close="isActive = false"
+    >
+      <ec-btn
+        :is-disabled="isDisabled"
+        is-reverse
+        is-rounded
+        category="primary"
+        :class="{
+          'ec-btn-dropdown__dropdown-btn': true,
+          'ec-btn-dropdown__dropdown-btn--active': isActive,
+        }"
+        data-test="ec-btn-dropdown__dropdown-btn"
+        icon="simple-arrow-drop-down"
+      />
+    </ec-dropdown-search>
+  </div>
+</template>
+
+<script>
+import EcBtn from '../ec-btn';
+import EcDropdownSearch from '../ec-dropdown-search';
+
+export default {
+  name: 'EcBtnDropdown',
+  components: {
+    EcBtn,
+    EcDropdownSearch,
+  },
+  props: {
+    itemsDropdown: {
+      type: Array,
+      default: () => [],
+    },
+    isDisabled: {
+      type: Boolean,
+      default: false,
+    },
+    textButton: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      isActive: false,
+    };
+  },
+  computed: {
+    popperModifiers() {
+      return {
+        setPopperWidth: {
+          enabled: true,
+          order: 840,
+          fn: /* istanbul ignore next */ (data) => {
+            data.styles.width = this.$refs.popperReference.offsetWidth;
+            return data;
+          },
+        },
+        calculatePopperPosition: {
+          enabled: true,
+          order: 845,
+          fn: /* istanbul ignore next */ (data) => {
+            const ref = this.$refs.popperReference;
+            data.offsets.popper.left = ref.offsetLeft;
+            data.popper.left = ref.offsetLeft;
+
+            return data;
+          },
+        },
+      };
+    },
+  },
+};
+</script>
+
+<style>
+  .ec-btn-dropdown {
+    @apply tw-flex;
+
```

Comment:
Please add
```suggestion
@apply tw-justify-center
```
We need it centered